index.js

```diff
@@ -171,10 +172,13 @@ export const Rels = {
 export const Classes = {
 	activities: {
 		assigned: 'assigned',
+		assignmentActivity: 'assignment-activity',
```

Comment:
Out of curiosity, how are these used? I would have expected just `assignment` to be sufficient, but I'm probably not thinking of the right use case!

Reply:
They are used to tell the difference between the activity type (assignment vs quiz vs discussion/topic). I believe the `-activity` on the classes is to be explicit out-of-context.

Reply:
Should `out-of-context` or something just be a different class then? So like: `classes: ['assignment', 'activity','out-of-context']` or whatnot?